qbpm.platypus

```diff
@@ -0,0 +1,259 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>AcceptsFiles</key>
+	<true/>
+	<key>AcceptsText</key>
+	<false/>
+	<key>Authentication</key>
+	<false/>
+	<key>Author</key>
+	<string>Matthew Toohey</string>
+	<key>BundledFiles</key>
+	<array/>
+	<key>Creator</key>
+	<string>Platypus-5.3</string>
+	<key>DeclareService</key>
+	<false/>
+	<key>Destination</key>
+	<string>/Applications/qbpm.app</string>
+	<key>DevelopmentVersion</key>
+	<false/>
+	<key>DocIconPath</key>
+	<string></string>
+	<key>Droppable</key>
+	<true/>
+	<key>ExecutablePath</key>
+	<string>/usr/local/share/platypus/ScriptExec</string>
+	<key>Identifier</key>
+	<string>org.qbpm.qbpm</string>
+	<key>InterfaceType</key>
+	<string>None</string>
+	<key>InterpreterArgs</key>
+	<array/>
+	<key>InterpreterPath</key>
+	<string>/bin/sh</string>
+	<key>Name</key>
+	<string>qbpm</string>
+	<key>NibPath</key>
+	<string>/usr/local/share/platypus/MainMenu.nib</string>
+	<key>OptimizeApplication</key>
+	<true/>
+	<key>Overwrite</key>
+	<false/>
+	<key>PromptForFileOnLaunch</key>
+	<true/>
```

Comment:
You're right, that seems to have been a mistake, that behaviour wasn't intended (fixed with 32539f4d). I'm somewhat confused by the alternative behaviour as well though: I would expect it to run `qbpm choose` with no arguments, which from the command line results in a choice between profiles to open a new window for, but instead, it just does nothing and the application closes immediately. I'm not really sure what's going on, lmk if you've got any ideas, but otherwise that behaviour is better than the file picker.